diff --git a/rofi-wifi-menu.sh b/rofi-wifi-menu.sh
index fb45671..c5ab108 100755
--- a/rofi-wifi-menu.sh
+++ b/rofi-wifi-menu.sh
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash

-notify-send "Getting list of available Wi-Fi networks..."
+# notify-send "Getting list of available Wi-Fi networks..." # Uncomment to enable notifications
 # Get a list of available wifi connections and morph it into a nice-looking list
-wifi_list=$(nmcli --fields "SECURITY,SSID" device wifi list | sed 1d | sed 's/  */ /g' | sed -E "s/WPA*.?\S/ /g" | sed "s/^--/ /g" | sed "s/  //g" | sed "/--/d")
+wifi_list=$(nmcli --fields "SECURITY,SSID" device wifi list | sed 1d | sed 's/  */ /g' | sed -E "s/WPA*.?\S/ /g" | sed "s/^--/ /g" | sed "s/  //g" | sed "/--/d" | sort -u)

 connected=$(nmcli -fields WIFI g)
 if [[ "$connected" =~ "enabled" ]]; then
